spec.bs

```diff
@@ -1236,8 +1236,9 @@ Set local testing mode {#set-local-testing-mode}
     </table>
 </figure>
 
-<div algorithm>
-The <dfn export>remote end steps</dfn> are:
+<div algorithm="remote end steps">
+The <a spec="webdriver2">remote end steps</a> given <var ignore>session</var>,
+[=URL=] <var ignore>variables</var> and |parameters| are:
```

Comment:
is it needed to link to URL, or simply <var ignore>URL variables</var>? It seems "URL variables" is the argument's name, e.g., https://w3c.github.io/webdriver/#find-elements-from-element

Reply:
Ah, good catch, thanks! I was confused by the space, but you're right.